app/jobs/mfa_usage_stats_job.rb

```diff
@@ -2,14 +2,14 @@ class MfaUsageStatsJob < ApplicationJob
   queue_as "stats"
 
   def perform
-    non_mfa_users = User.where(mfa_seed: nil).where.not(id: WebauthnCredential.select(:user_id)).count
-    otp_only_users = User.where.not(mfa_seed: nil).where.not(id: WebauthnCredential.select(:user_id)).count
-    webauthn_only_users = User.where(mfa_seed: nil).where(id: WebauthnCredential.select(:user_id)).count
-    webauthn_and_otp_users = User.where.not(mfa_seed: nil).where(id: WebauthnCredential.select(:user_id)).count
+    non_mfa_users = User.where(totp_seed: nil).where.not(id: WebauthnCredential.select(:user_id)).count
+    totp_only_users = User.where.not(totp_seed: nil).where.not(id: WebauthnCredential.select(:user_id)).count
+    webauthn_only_users = User.where(totp_seed: nil).where(id: WebauthnCredential.select(:user_id)).count
+    webauthn_and_totp_users = User.where.not(totp_seed: nil).where(id: WebauthnCredential.select(:user_id)).count
 
     StatsD.gauge("mfa_usage_stats.non_mfa_users", non_mfa_users)
-    StatsD.gauge("mfa_usage_stats.otp_only_users", otp_only_users)
+    StatsD.gauge("mfa_usage_stats.totp_only_users", totp_only_users)
```

Comment:
Any thoughts on this change? If we do this rename, we'll need to create a new dashboard that tracks metrics against `mfa_usage_stats.totp_only_users`. 

I'm not sure whether we can merge the stats collected from tracking `mfa_usage_stats.otp_only_users` in addition to the new metrics that will be named `mfa_usage_stats.totp_only_users`.

Since it's internal to us, we don't necessarily _need_ to make this change. The downside is we're diverging from consistent terminology, which I don't like. 

In practicality, if it turns out it's not possible to merge both datasets in one dashboard, I don't think it's a huge deal. We only started tracking this metric ~2 months ago iirc. We can keep the existing dashboard and create a new one for tracking under `mfa_usage_stats.totp_only_users` moving forward.